dashboard/src/main/resources/templates/component.ftl

```diff
@@ -1,8 +1,11 @@
+<!DOCTYPE html>
 <html lang="en-US">
   <#include "macros.ftl">
   <head>
+    <meta charset="utf-8" />
```

Comment:
Without encoding specification, the multi-byte character "▶" was not shown correctly.

Although Google HTML style says [do not  close void elements](https://google.github.io/styleguide/htmlcssguide.html#Document_Type), meta tags need to close in order to be parsed by xom document parser for unit tests.